### PR TITLE
Branch

### PR DESCRIPTION
**Pull request by ghost**
made modifications because jonathan wanted me to

### CODE REVIEW

File: capture_the_flag/app.py [hunk 1]
```diff
@@ -1,7 +1,10 @@
 import flask
 import flask_login
-from flask_restful import Api
+import os
+
+os.chdir('<parent-directory>')
```
Comment (jtschoonhoven) [inline capture_the_flag/app.py]: This breaks the app.

**Review by jtschoonhoven — CHANGES_REQUESTED** [review]
The imports are organized according to PEP8. Please don't change them without cause.
https://www.python.org/dev/peps/pep-0008/#imports

**Comment by ghost** [comment]
made requested changes

File: capture_the_flag/app.py [hunk 2]
```diff
@@ -1,11 +1,10 @@
 import flask
 import flask_login
-from flask_restful import Api
-
 from capture_the_flag import database as db
 from capture_the_flag.models.user import load_user_from_session, User
 from capture_the_flag.session import SuperSecureSessionInterface
 from capture_the_flag.routes import files, index, login, logout, signup, users
+from flask_restful import Api
```
Comment (jtschoonhoven) [inline capture_the_flag/app.py]: please put this back with the other 3rd party imports

File: capture_the_flag/database.py [hunk 3]
```diff
@@ -1,5 +1,4 @@
 import sqlite3  # noqa
-
```
Comment (jtschoonhoven) [inline capture_the_flag/database.py]: >You should put a blank line between each group of imports.

please don't delete newlines between groups of imports

File: capture_the_flag/models/base.py [hunk 4]
```diff
@@ -1,7 +1,5 @@
 from datetime import datetime
-
```
Comment (jtschoonhoven) [inline capture_the_flag/models/base.py]: > You should put a blank line between each group of imports.

please don't delete newlines between groups of imports

File: capture_the_flag/models/user.py [hunk 5]
```diff
@@ -1,10 +1,7 @@
 import random
 import time
-
```
Comment (jtschoonhoven) [inline capture_the_flag/models/user.py]: > You should put a blank line between each group of imports.

please don't delete newlines between groups of imports

File: capture_the_flag/routes/files.py [hunk 6]
```diff
@@ -1,5 +1,4 @@
 import os
-
```
Comment (jtschoonhoven) [inline capture_the_flag/routes/files.py]: > You should put a blank line between each group of imports.

please don't delete newlines between groups of imports

File: requirements.txt [hunk 7]
```diff
@@ -1,4 +1,6 @@
+#Python 2.7
```
Comment (jtschoonhoven) [inline requirements.txt]: This should go in `setup.py`, not requirements.txt. The requirements themselves are not specific to any Python version.

**Review by jtschoonhoven — CHANGES_REQUESTED** [review]
https://www.python.org/dev/peps/pep-0008/#imports

> Imports should be grouped in the following order:
> 
> 1. standard library imports
> 2. related third party imports
> 3. local application/library specific imports
> 
> You should put a blank line between each group of imports.

**Comment by jtschoonhoven** [comment]
Also, please don't check in `database.db`

**Comment by ghost** [comment]
yes. definitely

**Pull request closed without merging**